nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -123,6 +125,7 @@ def __init__(
         untie_biases=True,
         pos_emb_max_len=5000,
         conv_kernel_size=31,
+        conv_norm_type='batch_norm',
         dropout=0.1,
```

Comment:
So this is backwards compatible ?

Reply:
Yes, the default would be batch_norm which has been used for older models.